examples/mfem/bp1.hpp

```diff
@@ -115,8 +115,8 @@ class CeedMassOperator : public mfem::Operator {
     const int ir_order = 2*(order + 2) - 1; // <-----
     const mfem::IntegrationRule &ir =
       mfem::IntRules.Get(mfem::Geometry::SEGMENT, ir_order);
-    CeedInt nqpts, nelem = mesh->GetNE(), dim = mesh->SpaceDimension(),
-            ncompx = dim;
+    CeedInt nelem = mesh->GetNE(), dim = mesh->SpaceDimension(),
```

Comment:
When there is a mix of declarations with and without initializations, astyle indents correctly when the initializations come first.